core/variant/variant.h

```diff
@@ -123,7 +125,9 @@ class Variant {
 		PACKED_FLOAT64_ARRAY,
 		PACKED_STRING_ARRAY,
 		PACKED_VECTOR2_ARRAY,
+		PACKED_VECTOR2I_ARRAY,
 		PACKED_VECTOR3_ARRAY,
+		PACKED_VECTOR3I_ARRAY,
```

Comment:
These should be added to the end of the list to avoid unnecessary compatibility breaking.